Master Project/Assets/Scenes/Combat/Qi_Scripts/MonsterAction.cs

```diff
@@ -50,6 +52,31 @@ void Awake()
             Ih = GameObject.Find("InfoHider");
             Tk = GameObject.Find("Talk").GetComponent<Talk>();
             Ad = GameObject.Find("Monster").GetComponent<AudioController>();
+            if (GameObject.Find("Nessie"))
+            {
+                Nessie = GameObject.Find("Nessie");
+            }
+            else
+            {
+                Debug.Log("Cant Find Nessie");
+            }
+
+            if (GameObject.Find("Cerberus"))
+            {
+                Cerberus = GameObject.Find("Cerberus");
+            }
+            else
+            {
+                Debug.Log("Cant Find Cerberus");
+            }
+            if (GameObject.Find("REDACTED"))
+            {
+                REDACTED = GameObject.Find("REDACTED");

```

Comment:
I'm not sure if this how Siān wants the name to be printed (it may be "[REDACTED]"). Can we move all three of these names to a constant somewhere if we're going to do this name based method? Then we can make an easy change if ever necessary.

Reply:
How about I save those names as a list?